src/app/docs/model.html

```diff
@@ -36,6 +36,8 @@ <h1>
                 <li ui-sref-active='active'><a ui-sref="dbt.model({'#': 'details'})">Details</a></li>
                 <li ui-sref-active='active'><a ui-sref="dbt.model({'#': 'description'})">Description</a></li>
                 <li ui-sref-active='active'><a ui-sref="dbt.model({'#': 'columns'})">Columns</a></li>
+                <li ui-sref-active='active'><a ui-sref="dbt.model({'#': 'referenced_by'})">Referenced By</a></li>
```

Comment:
We can and should figure out a better way to reduce the copy/paste necessary for these different node types. Not a problem at all for right now though :)

Reply:
Sounds good! Let me know if you would like me to spend some time on figuring out how to reduce the copy pasting.